README.md

```diff
@@ -230,8 +222,8 @@ to publish their data model files for debugging/viewing:
 ### Push to Your Simplifier Project
 Publish both the conformance resources and example resources to Simplifier
 ```
-fhirmodel publish ./site_root/source/resources --username=$SIMPLIFIER_USER --password=$SIMPLIFIER_PW --base_url=<your server>
-fhirmodel publish ./site_root/source/examples --username=$SIMPLIFIER_USER --password=$SIMPLIFIER_PW --base_url=<your server>s
+fhirmodel publish ./site_root/input/resources/profiles --username=$SIMPLIFIER_USER --password=$SIMPLIFIER_PW --base_url=<your server>
+fhirmodel publish ./site_root/input/resources/examples --username=$SIMPLIFIER_USER --password=$SIMPLIFIER_PW --base_url=<your server>s
```

Comment:
Is the `'s'` at the end of the line typo?

Reply:
Yep 👍